pom.xml

```diff
@@ -5,7 +5,7 @@
 
   <groupId>com.replyyes</groupId>
   <artifactId>facebook-messenger</artifactId>
-  <version>0.0.4-SNAPSHOT</version>
+  <version>0.0.5-SNAPSHOT</version>
```

Comment:
Sorry, should have been more clear here
So this should be 0.1.0 for the build. We are currently at 0.0.3 in PROD and 0.0.4-SNAPSHOT in DEV

This change going to change method signatures so technically we should do at least a minor version update since it is going to break existing builds.

Once the changes are deployed and built, we should move the version to 0.1.1-SNAPSHOT